set.mm

```diff
@@ -512846,6 +512945,25 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
       BCDEFGHIJKTUA $.
   $}
 
+  ${
+    ringdi22.1 $e |- B = ( Base ` R ) $.
+    ringdi22.2 $e |- .+ = ( +g ` R ) $.
+    ringdi22.3 $e |- .x. = ( .r ` R ) $.
+    ringdi22.4 $e |- ( ph -> R e. Ring ) $.
+    ringdi22.5 $e |- ( ph -> X e. B ) $.
+    ringdi22.6 $e |- ( ph -> Y e. B ) $.
+    ringdi22.7 $e |- ( ph -> Z e. B ) $.
+    ringdi22.8 $e |- ( ph -> T e. B ) $.
+    $( Develop the product of two sums in a ring.  (Contributed by Thierry
```

Comment:
I think "Expand the product" is a better term.

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4872/commits/cc2150ead40d9ee3851e56d8f37942b85b152b14